pkg/fleetautoscalers/controller.go

```diff
@@ -366,7 +366,12 @@ func (c *Controller) updateStatus(ctx context.Context, fas *autoscalingv1.FleetA
 
 	if !apiequality.Semantic.DeepEqual(fas.Status, fasCopy.Status) {
 		if scalingLimited {
-			c.recorder.Eventf(fas, corev1.EventTypeWarning, "ScalingLimited", "Scaling fleet %s was limited to maximum size of %d", fas.Spec.FleetName, desiredReplicas)
+			// scalingLimited indicates that the calculated scale would be above or below the range defined by MinReplicas and MaxReplicas
+			if currentReplicas > desiredReplicas {
+				c.recorder.Eventf(fas, corev1.EventTypeWarning, "ScalingLimited", "Scaling fleet %s was limited to maximum size of %d", fas.Spec.FleetName, desiredReplicas)
+			} else {
+				c.recorder.Eventf(fas, corev1.EventTypeWarning, "ScalingLimited", "Scaling fleet %s was limited to minimum size of %d", fas.Spec.FleetName, desiredReplicas)
+			}
```

Comment:
```suggestion
      msg := "Scaling fleet %s was limited to minimum size of %d"
			if currentReplicas > desiredReplicas {
         msg := "Scaling fleet %s was limited to maximum size of %d"
			} 
			
			c.recorder.Eventf(fas, corev1.EventTypeWarning, "ScalingLimited", msg, fas.Spec.FleetName, desiredReplicas)
```

WDYT? Remove some duplication of strings?

Reply:
Yes make sense